screens/Home.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import {
+  ImageBackground,
+  Text,
+  Button,
+  StyleSheet,
+  View,
+  TextInput,
+} from 'react-native'
+import styled from 'styled-components/native'
+
+const image = {
+  uri:
+    'https://images.unsplash.com/photo-1615224572756-729e7d1cd942?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=934&q=80',
+}
+
+const HomeContainer = styled.View`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vw;
```

Comment:
![image](https://user-images.githubusercontent.com/50926159/115146080-572a8200-a055-11eb-9370-fbf6139bb8c7.png)

Reply:
I changed the height property and now I'm able to run the project on my device.